packages/project-integrity/README.md

```diff
@@ -0,0 +1,23 @@
+# @sap-ux/project-integrity
+
+Library that offers functions to detect modifications to a set of files within a project. It also allows to pass addition key-> string values that are stored together with integrity data and can also be checked.
+For specific Fiori project, this module contains convenience functions that take care of building up the file and additional string content list, consumer need just to pass a valid Fiori project root.
+
+## Installation
+Npm   
+`npm install --save @sap-ux/project-access`
```

Comment:
Is that a copy/paste error because it uses project-access all over the place in readme?

Reply:
Yes, thanks for pointing it out, I've fixed it:
https://github.com/SAP/open-ux-tools/pull/2630/commits/57021113a9185e81d0c49bce46f57aa1dce2e2f7